### ISSUE TITLE
python interface

### ISSUE THREAD

**Issue by fixerslo** [LinuxCNC/linuxcnc#180]
1. When in MDI mode, `linuxcnc.stat.settings[1]`  (Feed value) returns wrong value if in G95 mode after spindle is started:

-Issue G95 in MDI
-set feed with F word
-**go to jog mode and back to MDI**
-turn on spindle with M3 S

## This is what I expected to happen:
1. `linuxcnc.stat.settings[1]` should return the value that was set with F word

## This is what happened instead:
1.  `linuxcnc.stat.settings[1]` returns value, that is 60 times greater than value that was set with F.  

## Information about my hardware and software:

 LinuxCNC 2.7.4 
Gmoccapy GUI
metric mode

EDIT: typo

**Comment by SebKuzminsky** [comment]
I can replicate this problem with LinuxCNC 2.7.7.

I started sim/axis/axis, F1 F2 Ctrl-Home, switch to the MDI tab, G21, G95,
F10 (10 mm/rev).

Then in another window I ran python, imported the linuxcnc module, instantiated linuxcnc.stat, polled, and got settings = (0.0, 10.0, 0.0).  Back in Axis I switched to the Manual tab, jogged around a little, switched back to MDI and ran 'S1000 M3'.  Back in python i polled linuxcnc.stat again and got settings = (0.0, 600.0, 1000.0).  In Axis MDI tab, the 'Active G-Codes' also shows F600.
